src/cmd/manage_nsfs.js

```diff
@@ -259,6 +266,63 @@ async function delete_bucket(data, force) {
     }
 }
 
+
+/**
+ * filter_bucket will return true or false based on whether a bucket meets the criteria defined by the supported flags.
+ * currently not implemented
```

Comment:
?

Reply:
You're right, but please notice that I just moved this.
It was added in PR #7745 - do you want me to remove the " * currently not implemented" comment?